clients/sfizz_jack.man.in

```diff
@@ -1,31 +1,34 @@
-.TH sfizz_jack 1 "@MAN_TODAY@" "@CMAKE_PROJECT_VERSION@" "sfizz_jack man page"
+.TH sfizz_jack 1 "@MAN_TODAY@" "@CMAKE_PROJECT_VERSION@" "sfizz"
 .SH NAME
-sfizz_jack \- Launch a sfizz instance as a JACK client
+sfizz_jack \- launch a sfizz instance as a JACK client
 .SH SYNOPSIS
-sfizz_jack [FILE] [OPTIONS...]
+.B sfizz_jack
+[\fIFILE\fR] [\fIOPTIONS\fR...]
```

Comment:
`\fI` is "start italic" and `\fR` is "start regular"?

Reply:
Yes. Instead of \fR I believe we can also say \fP (previous font, probably; I’ll need to dig out the manual to be sure), which undoes the most recent font change.